extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -193,9 +190,20 @@ export class DoubleclickA4aEligibility {
       addExperimentIdToElement(experimentId, element);
       forceExperimentBranch(win, DOUBLECLICK_A4A_EXPERIMENT_NAME, experimentId);
     }
-    return ![DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_CONTROL,
+    // If we are in the Remote.html or USDRUD controls, then we are supposed
+    // to use Delayed Fetch in accordance with the existing carveouts.
+    if ([DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_CONTROL,
```

Comment:
I think this can be:

return (!hasUSDRD || experimentId == DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT) && (!useRemoteHtml || experimentId == DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT);